test-container/src/main/java/io/strimzi/StrimziKafkaContainer.java

```diff
@@ -20,7 +20,9 @@
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class StrimziKafkaContainer extends GenericContainer<StrimziKafkaContainer> {
```

Comment:
Can we add a sentence of two of Javadoc explaining what this is and how it's intended to be used?